src/api/entityResolution/types.ts

```diff
@@ -279,3 +279,59 @@ export type IngestionStatus =
        */
       timeLeftSeconds?: number;
     };
+
+/**
+ *  Informations about the entity resolution license, across all data-sources.
+ */
+export interface EntityResolutionLicenseInfo {
+  /**
+   * The number of records ingested.
+   */
+  recordsIngested: number;
+  /**
+   * The number of records available in the license.
+   */
+  recordsAvailable: number;
+}
+
+/**
+ *  The entity resolution metrics, for a given data-source.
+ */
+export interface EntityResolutionMetrics {
+  /**
+   * Temporal information about the last ingestion task. Undefined if ingestion has never run.
+   */
+  lastIngestion?: {
```

Comment:
`lastIngestion` makes it clear this only relates to ingestion tasks, not purge tasks.